readme.md

```diff
@@ -106,7 +126,26 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+// GET method route
+app.get('/', function (req, res) {
+  res.send('GET request to the homepage');
+});
+
+// POST method route
+app.post('/', function (req, res) {
+  res.send('POST request to the homepage');
+});
+
+// DELETE method route
+app.delete('/', function (req, res) {
+  res.send('DELETE request to homepage');
```

Comment:
:+1:  You're correct, but something to keep in mind - where do we normally send `put/patch/delete` requests?

Maybe something like:
```
app.put("/:id", function(request, response){
  response.send("UPDATE");
});
app.delete("/:id", function(request, response){
  response.send("DELETE");
});
```